docs/src/pages/[platform]/components/link/index.page.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Link
-description: Links are customizable and themable elements used for Navigation. By default Links render an anchor tag but can be configured to be used with routing libraries.
+description: Links help users navigate within an application. Links render an anchor element <a> by default, and can also be used with routing libraries. 
```

Comment:
For a brief description, I think we can make it simple - `Links semantically renders an <a> and are primarily used for navigation.` Thoughts?